include/seqan3/io/alignment_file/format_bam.hpp

```diff
@@ -394,7 +395,7 @@ inline void format_bam::read_alignment_record(stream_type & stream,
         ref_id = core.refID;                                                   // field::REF_ID
     }
 
-    flag = core.flag;                                                          // field::FLAG
+    flag = sam_flag{static_cast<uint16_t>(core.flag)};                         // field::FLAG
```

Comment:
why is core.flag not already an uin16_t?

Reply:
Because I copied the core structure from seqan2 aand I did not know that using bit fields is obsolete if you use the correct integers...